package.json

```diff
@@ -3,5 +3,13 @@
 		"dev:android": "npx concurrently -n \"Android,Server\" \"npm run run:android\" \"npm run run:server\"",
 		"run:android": "cd react-native-app && npm run android",
 		"run:server": "cd backend && npm run dev"
+	},
+	"dependencies": {
+		"amazon-cognito-identity-js": "^6.1.2",
+		"aws-sdk": "^2.1317.0",
+		"jsonwebtoken": "^9.0.0",
+		"jwk-to-pem": "^2.0.5",
+		"node-fetch": "^3.3.0",
```

Comment:
I completely forgot about these. These were part of an early effort to implement Cognito through the front-end before moving it to the backend. I forgot I had added these so I forgot to delete them. Also I always forget to cd into react-native-app or backend before installing so that is why it was added to the root on accident.